grails-app/utils/org/pih/warehouse/NullBinderHelper.groovy

```diff
@@ -0,0 +1,12 @@
+package org.pih.warehouse
+
+import grails.databinding.BindingHelper
+import grails.databinding.DataBindingSource
+
+class NullBinderHelper<T> implements BindingHelper<T> {
```

Comment:
I think we need to be more explicit with the naming of the binder like for example `EmptyStringsToNullBinder` so that we know exactly what is it doing